mysql-test/suite/tianmu/t/out_of_range_issue1151.test

```diff
@@ -0,0 +1,186 @@
+--source include/have_tianmu.inc
+
+drop database if exists test;
+create database test;
```

Comment:
Please use database such as out_of_range_issue1151 instead of test

Reply:
Fixed.